process_stats.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+import sys
+import json
+from prettytable import PrettyTable
+from github import Github
+import os
+
+file_name = sys.argv[1]
+
+with open(file_name) as f:
+    data = json.load(f)
+
+metrics = {}
+
+for point in data:
+    metrics[point['metric'] + "-" + point['measurement']] = point['value']
+
+METRICS_TO_REPORT = ['READ-AverageLatency(us)',
+                     'READ-50thPercentileLatency(us)',
+                     'READ-99thPercentileLatency(us)',
+                     'READ-MaxLatency(us)',
+                     'UPDATE-AverageLatency(us)',
+                     'UPDATE-50thPercentileLatency(us)',
+                     'UPDATE-99thPercentileLatency(us)',
+                     'UPDATE-MaxLatency(us)']
+
+table = PrettyTable()
+table.field_names = ["Metric", "Value"]
+
+for key, value in metrics.items():
+    if key in METRICS_TO_REPORT:
+        table.add_row([key, value])
+
+pr_comment = "YCSB Test Results :fire::fire::fire::fire:"
+pr_comment += "\n"
+pr_comment += "```"
+pr_comment += "\n"
```

Comment:
Would it be good to wrap these into a function and do something like this?
```python
def process(file_name):
    xxx
   
if __name__ == '__main__':
    process(sys.argv[1])
```
I thought it could guard this file in future imports if there would be any